diff --git a/config.mk b/config.mk
index 470bbe6..2c1e1ac 100644
--- a/config.mk
+++ b/config.mk
@@ -4,12 +4,12 @@ PREFIX ?= /usr
 FRIDA := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
 # Features ordered by binary footprint, from largest to smallest
-FRIDA_V8 ?= auto
+FRIDA_V8 ?= disabled
 FRIDA_CONNECTIVITY ?= enabled
 FRIDA_DATABASE ?= enabled
-FRIDA_JAVA_BRIDGE ?= auto
-FRIDA_OBJC_BRIDGE ?= auto
-FRIDA_SWIFT_BRIDGE ?= auto
+FRIDA_JAVA_BRIDGE ?= enabled
+FRIDA_OBJC_BRIDGE ?= disabled
+FRIDA_SWIFT_BRIDGE ?= disabled
 
 # Include jailbreak-specific integrations
 FRIDA_JAILBREAK ?= auto
